.github/workflows/bug-hunter.md

```
@@ -0,0 +1,304 @@
+return parse(in);
+```
+
+**After**
+
+```java
+try (InputStream in = Files.newInputStream(path)) {
+  return parse(in);
+}
+```
+
+### Preserve a valid zero value
```

markdownlint-cli2 (0.23.2)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)